browser-build.config.js

```diff
@@ -0,0 +1,51 @@
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+ module.exports = {
+  entry: "./src/index.js",
+  output: {
+      filename: "geostyler.js",
+      path: __dirname + "/browser",
+      library: 'GeoStyler'
+  },
+   // Enable sourcemaps for debugging webpack's output.
+  devtool: "source-map",
+   resolve: {
+      // Add '.ts' and '.tsx' as resolvable extensions.
+      extensions: [".ts", ".tsx", ".js", ".json"]
+  },
+   module: {
+      rules: [
+          {
+            test: /\.css$/,
+            use: ExtractTextPlugin.extract({
+              fallback: "style-loader",
+              use: "css-loader"
+            })
+          },
+          // All files with a '.ts' or '.tsx' extension will be handled by 'awesome-typescript-loader'.
+          {
+            test: /\.tsx?$/,
+            loader: "awesome-typescript-loader"
+          },
+          // locale js files need to be transpiled to es5
+          {
+            test: /\.js?$/,
+            include: /src\/locale/,
+            loader: 'babel-loader',
+            query: {
+              presets: ['@babel/env']
+            }
+          }
+      ]
+  },
+   plugins: [
+    new ExtractTextPlugin("geostyler.css")
+  ],
```

Comment:
Is there also a default optimization config in older webpack versions (<4) present? E.g. for `UglifyJS`?